tests/ui/feature-gates/feature-gate-precise_pointer_size_matching.rs

```diff
@@ -1,17 +1,17 @@
 fn main() {
     match 0usize {
-        //~^ ERROR non-exhaustive patterns: `_` not covered
-        //~| NOTE pattern `_` not covered
+        //~^ ERROR non-exhaustive patterns: `usize::MAX..` not covered
+        //~| NOTE pattern `usize::MAX..` not covered
```

Comment:
It may be better to output `usize::MAX + 1` here.

Reply:
Yeah, that's the "clear enough" I was mentioning above. I feels complicated to output that. I'd have to special-case `thir::PatRange` for that probably, because I can't store `usize::MAX+1` in a `Const` of type `usize`. I was hoping the overlap was irrelevant

Reply:
Plus that's a not valid rust pattern, I'd need to output `const {usize::MAX+1}..` which is unstable